### ISSUE TITLE
Windows Magnifier doesn't follow text cursor

### ISSUE THREAD

**Issue by errestan** [microsoft/terminal#9243]
The Windows Magnifier application does not follow the text cursor in the Windows Terminal when the text cursor follow option is enabled in Windows Magnifier. This also affects the ZoomText third party screen magnification software.

**Comment by DHowett** [comment]
I could have sworn we had another copy of this in our issue tracker. Thanks for the report!

**Comment by zadjii-msft** [comment]
I suspect this was also fixed by #12210, which should be in 1.13
